diesel_tests/tests/types.rs

```diff
@@ -380,6 +384,34 @@ fn pg_numeric_from_sql() {
 
 #[test]
 #[cfg(feature = "postgres")]
+fn pg_numeric_bigdecimal_to_sql() {
+    use self::bigdecimal::BigDecimal;
+
+    fn correct_rep(integer: u64, decimal: u64) -> bool {
+        let expected = format!("{}.{}", integer, decimal);
+        let value: BigDecimal = expected.parse().expect("Could not parse to a BigDecimal");
+        query_to_sql_equality::<Numeric, BigDecimal>(&expected, value)
+    }
+
+    quickcheck(correct_rep as fn(u64, u64) -> bool);
+}
+
+#[test]
+#[cfg(feature = "postgres")]
+fn pg_numeric_bigdecimal_from_sql() {
+    use self::bigdecimal::BigDecimal;
+
+    let query = "1.0::numeric";
+    let expected_value: BigDecimal = "1.0".parse().expect("Could not parse to a BigDecimal");
+    assert_eq!(expected_value, query_single_value::<Numeric, BigDecimal>(query));
+
+    let query = "141.00::numeric";
+    let expected_value: BigDecimal = "141.00".parse().expect("Could not parse to a BigDecimal");
+    assert_eq!(expected_value, query_single_value::<Numeric, BigDecimal>(query));
+}
```

Comment:
Do you want to add some more extreme tests? To tests values that exceed `u64` and `f64`?